wire-runtime/src/main/java/com/squareup/wire/ProtoAdapter.java

```diff
@@ -66,6 +67,18 @@ public ProtoAdapter(FieldEncoding fieldEncoding, Class<?> javaType) {
     return new RuntimeEnumAdapter<>(type);
   }
 
+  /**
+   * Creates a new proto adapter for a map using {@code keyAdapter} and {@code valueAdapter}.
+   * <p>
```

Comment:
For what it’s worth, putting `<p>` on its own line like this makes CMD+ALT+L work less well.